src/main/java/com/eleks/academy/whoami/core/state/StartAssignCharacters.java

```diff
@@ -0,0 +1,111 @@
+package com.eleks.academy.whoami.core.state;
+
+import com.eleks.academy.whoami.core.SynchronousPlayer;
+import com.eleks.academy.whoami.core.exception.GameException;
+import com.eleks.academy.whoami.enums.GameStatus;
+import com.eleks.academy.whoami.model.request.CharacterSuggestion;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public final class StartAssignCharacters extends AbstractGameState {
```

Comment:
Call it just "Start" because we already have "SuggestCharacters" now "StartAssignCharacters" too confusing

Reply:
Resolved
Funny that firstly I name it Start but I think that this name has a small meaning.